legate/core/legion.py

```diff
@@ -5043,7 +5050,7 @@ def pack_dtype(self, dtype):
     def get_string(self):
         if self.string is None or self.arglen != len(self.args):
             fmtstr = "".join(self.fmt)
-            assert len(fmtstr) == len(self.args) + 1
+            #assert len(fmtstr) == len(self.args) + 1
```

Comment:
This assertion is being removed for performance reasons. The new method `pack_32bit_int_arr` above is a much faster way of packing a set of 32 bit ints (rather than appending them individually), but it breaks the invariant that `len(fmtstr) == len(self.args) + 1`

Reply:
I'll probably generalize this code, as I'm sure this is a utility not just for 32-bit integers.

Reply:
makes sense; I found it faster to batch the packing  of ints this way.